### PR TITLE
fix: proxy worker trigger

### LINKED ISSUE #1713: Proxy worker. It's started after each operation.
## Description
Proxy-worker is started after all operations.

## Steps to Reproduce

1. Go to staking page
2. Make any operation (example - withdraw)
3. Check logs

## Expected Behavior
Perform withdraw operation, proxy-worker should not be run

## Actual Behavior
Proxy-worker was run after submit the operation

## Environment
https://github.com/novasamatech/nova-spektr/commit/34bc8d02b705548c958d21b2a6fd32d2ecb3a151

<details>
  <summary>🍿 Example 🍿</summary>

https://github.com/novasamatech/nova-spektr/assets/40560660/bf29208c-00c9-41b3-a8e7-f0320c67f402

</details>

### PR DESCRIPTION
**Pull request by sokolova-an**

### CODE REVIEW

File: src/renderer/features/operations/OperationSubmit/model/submit-model.ts [hunk 1]
```diff
@@ -35,7 +35,7 @@ const formSubmitted = createEvent<ExtrinsicResultParams>();
 const extrinsicSucceeded = createEvent<ExtrinsicResultParams>();
 const extrinsicFailed = createEvent<string>();
 
-const $submitStore = restore<Input>(formInitiated, null);
+const $submitStore = restore<Input>(formInitiated, null).reset(formSubmitted);
```
Comment (sokolova-an) [inline src/renderer/features/operations/OperationSubmit/model/submit-model.ts:38]: I've seen we have it for other common stores. Do we need it here as well in this case?

File: src/renderer/widgets/WalletDetails/model/wallet-provider-model.ts [hunk 2]
```diff
@@ -250,11 +248,6 @@ sample({
   target: $proxyForRemoval,
 });
```
Comment (sokolova-an) [inline src/renderer/widgets/WalletDetails/model/wallet-provider-model.ts:250]: I believe it's better to move to add proxy model

File: src/renderer/widgets/WalletDetails/model/wallet-provider-model.ts [hunk 3]
```diff
@@ -250,11 +248,6 @@ sample({
...
 
-sample({
-  clock: addProxyModel.output.flowFinished,
-  target: proxiesModel.events.workerStarted,
-});
-
```
Comment (sokolova-an) [inline src/renderer/widgets/WalletDetails/model/wallet-provider-model.ts:257]: I believe it's better to move to add proxy model

File: src/renderer/features/operations/OperationSubmit/model/submit-model.ts (hunk 1 above)
Comment (tuul-wq) [reply src/renderer/features/operations/OperationSubmit/model/submit-model.ts:38]: I guess this won't hurt anyone, 👍

**Review by tuul-wq — APPROVED** [review]

**Pull request merged**